examples/dcutr/src/main.rs

```diff
@@ -157,7 +157,6 @@ async fn main() -> Result<(), Box<dyn Error>> {
                     ..
                 })) => {
                     info!("Relay told us our public address: {:?}", observed_addr);
```

Comment:
```suggestion
                    info!("Relay told us our observed address: {:?}", observed_addr);
```

While we are at it. For future Max :P thinking like the past Max in the previous review.